sass/_base.scss

```diff
@@ -269,25 +198,17 @@ pre table {
     }
 
     .hero-block {
-
-        @media (max-width: 767px) {
-            margin: 60px 32px 70px;
-        }
-
-        margin: 89px 80px;
-
         h1 {
             font-style: normal;
             font-weight: 700;
-            font-size: 3.125rem;
+            font-size: 2.8rem;
```

Comment:
I wonder if these font sizes should be variables too or not. Mostly since we have some places (I think) where we have to reset this since for example the markdown stuff overwrote it, or we want markdown generated things to be one level smaller. So having a variable for that might be nice.

Though we might also just want to just use SCSS mixins for that instead. :thinking: